integrations/jest-plugin/src/index.ts

```diff
@@ -18,10 +21,12 @@ type RawSourceMap = ReturnType<typeof transform>["sourceMap"];
 export function process(
   src: string,
   filename: string,
+  options: TransformOptions<Options>,
 ): {code: string; map?: RawSourceMap | string | null} {
   const transforms = getTransforms(filename);
   if (transforms !== null) {
     const {code, sourceMap} = transform(src, {
+      ...options.transformerConfig,
```

Comment:
Could you put this one line lower so that it's possible to override `transforms`? It's still a little awkward if you need different transforms for different file extensions, but in most cases that's not necessary. That would enable a workaround for https://github.com/alangpierce/sucrase/issues/741 (at least in many cases) by omitting the `imports` transform when targeting ESM.